### ISSUE TITLE
Localized language support.

### ISSUE THREAD

**Issue by actrace** [openconnect/openconnect-gui#221]
### Description of the Issue
openconnect-gui is a nice GUI client. It's even better if you can support localized languages. I also want to be able to provide translation for localized languages.
ps. On Android, the openconnect app supports localized languages.

### Specifications
- **Version**: 1.5.2
- **Platform**: win64

**Comment by horar** [comment]
Yes, this is on in plan, just the code is in last year in cleanup/refactoring phase. Please stay tuned (for v1.6)
